__tests__/http/controllers/AccountTypesController.test.js

```diff
@@ -0,0 +1,171 @@
+const AccountTypes = require('@http/controllers/AccountTypesController')
+const AccountTypeService = require('@services/AccountTypesService')
+const httpMocks = require('node-mocks-http')
+const { StatusCodes } = require('http-status-codes')
+const Boom = require('@hapi/boom')
+
+AccountTypeService.create = jest.fn()
+AccountTypeService.update = jest.fn()
+AccountTypeService.fetchAll = jest.fn()
+AccountTypeService.fetchById = jest.fn()
+AccountTypeService.deleteById = jest.fn()
+
+const errorMessage = 'Error message'
+const BoomNotFoundError = Boom.notFound(errorMessage)
+const BoomBadDataError = Boom.badData(errorMessage)
+
+let req, res, next
+
+beforeEach(() => {
+  req = httpMocks.createRequest()
+  req.body = {}
+  res = httpMocks.createResponse()
+  next = jest.fn().mockImplementation(err => {
+    return Boom.isBoom(err)
+      ? res.status(err.output.statusCode).json(err.output.payload)
+      : res.status(err.status || 500).json({ message: err.message })
+  })
+})
+
+describe('Create account types', () => {
+  it('should have a create function', () => {
+    expect(typeof AccountTypes.create).toBe('function')
```

Comment:
do you care testing this in general? I wouldn't, also it gets tedious too. 

Try not to test all implementation details

Reply:
yep pointless